packages/react-router-website/modules/examples/PreventingUnload.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+  Prompt,
+  BrowserPrompt
+} from 'react-router-dom'
+
+const PreventingTransitionsExample = () => (
```

Comment:
Ah, yes. I missed that ;)

But ya, I'd prefer to leave examples out of the "fake browser" that rely on stuff in the real browser.